dotnet/targets/Xamarin.Shared.Sdk.targets

```
@@ -1022,6 +1048,39 @@
 		</ItemGroup>
 	</Target>
 
+	<Target Name="_CreateAOTDedupAssembly"
+			Condition="'$(_RunAotCompiler)' == 'true'">
```

This is necessary so that we don't try to do this when building on Windows without being connected to a Mac:

```suggestion
			Condition="'$(_RunAotCompiler)' == 'true' And '$(IsMacEnabled)' == 'true'">
```